README.md

```diff
@@ -2,61 +2,30 @@
 
```

Comment:
- [ ] Pull Request should have a summary describing introduced changes in more detail, kindly add a description to your PR so it reflects the changes on the PR

Reply:
Description added.